tests/components/risco/conftest.py

```diff
@@ -0,0 +1,148 @@
+"""Fixtures for Risco tests."""
+from unittest.mock import MagicMock, PropertyMock, patch
+
+from pytest import fixture
+
+from homeassistant.components.risco.const import DOMAIN, TYPE_LOCAL
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PIN,
+    CONF_PORT,
+    CONF_TYPE,
+    CONF_USERNAME,
+)
+
+from .util import TEST_SITE_NAME, TEST_SITE_UUID, zone_mock
+
+from tests.common import MockConfigEntry
+
+TEST_CLOUD_CONFIG = {
+    CONF_USERNAME: "test-username",
+    CONF_PASSWORD: "test-password",
+    CONF_PIN: "1234",
+}
+TEST_LOCAL_CONFIG = {
+    CONF_TYPE: TYPE_LOCAL,
+    CONF_HOST: "test-host",
+    CONF_PORT: 5004,
+    CONF_PIN: "1234",
+}
+
+
+@fixture
+def two_zone_cloud():
+    """Fixture to mock alarm with two zones."""
+    zone_mocks = {0: zone_mock(), 1: zone_mock()}
+    alarm_mock = MagicMock()
+    with patch.object(
+        zone_mocks[0], "id", new_callable=PropertyMock(return_value=0)
+    ), patch.object(
+        zone_mocks[0], "name", new_callable=PropertyMock(return_value="Zone 0")
+    ), patch.object(
+        zone_mocks[1], "id", new_callable=PropertyMock(return_value=1)
+    ), patch.object(
+        zone_mocks[1], "name", new_callable=PropertyMock(return_value="Zone 1")
+    ), patch.object(
+        alarm_mock,
+        "zones",
+        new_callable=PropertyMock(return_value=zone_mocks),
+    ), patch(
+        "homeassistant.components.risco.RiscoCloud.get_state",
+        return_value=alarm_mock,
+    ):
+        yield zone_mocks
+
+
+@fixture
+def options():
+    """Fixture for default (empty) options."""
+    yield {}
```

Comment:
We only need to yield if we're inside a context manager or if we want to tear down something after yielding.
```suggestion
    return {}
```